### PR TITLE
chore: x/oracle: clean up keys 

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

For some reason the x/oracle module's keys were given two different sets of variable names, so we're just using the `KeyPrefix*` set

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added appropriate labels to the PR
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by toteki — COMMENTED** [review]
Is it better to keep the ones that start with `KeyPrefix` or the ones that end in `Key`?

I tend to favor the first if they're being used as prefixes (and it also keeps their names grouped alphabetically)

File: x/oracle/types/keys.go [hunk 1]
```diff
@@ -21,45 +21,40 @@ const (
...
-	KeyPrefixFeederDelegation             = []byte{0x02} // prefix for each key to a feeder delegation
-	KeyPrefixMissCounter                  = []byte{0x03} // prefix for each key to a miss counter
-	KeyPrefixAggregateExchangeRatePrevote = []byte{0x04} // prefix for each key to a aggregate prevote
-	KeyPrefixAggregateExchangeRateVote    = []byte{0x05} // prefix for each key to a aggregate vote
-	ExchangeRateKey                       = []byte{0x01} // prefix for each key to a rate
-	FeederDelegationKey                   = []byte{0x02} // prefix for each key to a feeder delegation
-	MissCounterKey                        = []byte{0x03} // prefix for each key to a miss counter
-	AggregateExchangeRatePrevoteKey       = []byte{0x04} // prefix for each key to a aggregate prevote
-	AggregateExchangeRateVoteKey          = []byte{0x05} // prefix for each key to a aggregate vote
+	ExchangeRateKey                 = []byte{0x01} // prefix for each key to a rate
```
Comment (alexanderbez) [inline x/oracle/types/keys.go:24]: We should still keep the `KeyPrefix*` prefixes and remove the `Key` suffix IMO. This way, we're consistent with `x/leverage` as well.

Comment (alexanderbez) [reply x/oracle/types/keys.go:24]: It reflects the context of what it's used for. E.g. for errors that you'd wrap, it'd be `Err*`.

Comment (adamewozniak) [reply x/oracle/types/keys.go]: Yeah that's fair 👍

**Review by toteki — APPROVED** [review]

**Pull request merged**